charts/castai-agent/README.md

```diff
@@ -4,54 +4,54 @@ CAST AI agent deployment chart.
-| commonAnnotations | object | `{}` | Annotations to add to all resources. |
-| commonLabels | object | `{}` | Labels to add to all resources. |
-| containerSecurityContext.allowPrivilegeEscalation | bool | `false` |  |
-| containerSecurityContext.capabilities.drop[0] | string | `"ALL"` |  |
-| containerSecurityContext.readOnlyRootFilesystem | bool | `true` |  |
-| createNamespace | bool | `true` | Specifies whether a namespace should be created. |
-| dnsPolicy | string | `""` |  |
-| fullnameOverride | string | `"castai-agent"` |  |
-| hostNetwork.enabled | bool | `false` |  |
-| image.pullPolicy | string | `"IfNotPresent"` |  |
-| image.repository | string | `"us-docker.pkg.dev/castai-hub/library/agent"` |  |
-| nameOverride | string | `""` |  |
-| namespace | string | `"castai-agent"` |  |
-| nodeSelector | object | `{}` |  |
-| podAnnotations | object | `{}` | Annotations to be added to agent pods. |
-| podLabels | object | `{}` | Labels to be added to agent pods. |
-| priorityClass.enabled | bool | `true` |  |
-| priorityClass.name | string | `"system-cluster-critical"` |  |
-| provider | string | `""` | Name of the Kubernetes service provider one of: "eks", "gks", "aks", "kops". |
-| rbac | object | `{"enabled":true}` | Specifies whether RBAC Clusterrole should be created. |
-| replicaCount | int | `1` |  |
-| resources | object | `{}` |  |
-| securityContext.fsGroup | int | `1002` |  |
-| securityContext.runAsGroup | int | `1002` |  |
-| securityContext.runAsNonRoot | bool | `true` |  |
-| securityContext.runAsUser | int | `1002` |  |
-| securityContext.seccompProfile.type | string | `"RuntimeDefault"` |  |
-| serviceAccount.annotations | object | `{}` | Annotations to add to the service account. |
-| serviceAccount.create | bool | `true` | Specifies whether a service account should be created. |
-| serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template. |
-| tolerations | object | `{}` |  |
\ No newline at end of file
+| Key | Type | Default                                                                                                               | Description                                                                                                                                                                                      |
+|-----|------|-----------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
nitpick: not sure if this was needed, aligning such long rows don't make it any better